Week2/homework/index.js

```diff
@@ -0,0 +1,114 @@
+console.log(1);
+let HiIndo='Hallo, dunia!'
```

Comment:
Do you think it's better to use `let` or `const` here? 

And please use lowercase letter as the first letter for variables. So like this `hiIndo`. See [naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)

Reply:
Hi,@marcianoviereck!
Yes, i think it is better to use const here.
i have made the necessary changes. i hope it is ok now.
Thank you for your attention.